src/SourceBuild/content/prep.sh

```diff
@@ -157,6 +157,9 @@ function BootstrapArtifacts {
 if [ "$installDotnet" == true ]; then
   echo "  Installing dotnet..."
   (source ./eng/common/tools.sh && InitializeDotNetCli true)
+
+  # TODO: Remove once runtime dependency is gone (https://github.com/dotnet/runtime/issues/93666)
+  bash .dotnet/dotnet-install.sh --install-dir "$SCRIPT_ROOT/.dotnet" --version 8.0.0-preview.7.23375.6 --runtime dotnet
```

Comment:
Is the version number required?  The SDK should have the version it depends on hard coded.  Removing the version would be nice because then it doesn't need to be updated as the SDK is updated.

Reply:
That was a silly comment as we need to install the 8.0 version.  I guess what I am asking is if the full version is needed or if some variant of 8.0 or 8.0.0 can be specified.